src/main/java/com/company/Controller.java

```diff
@@ -1,14 +1,28 @@
 package com.company;
 
+import com.google.i18n.phonenumbers.NumberParseException;
+import com.google.i18n.phonenumbers.PhoneNumberUtil;
+import com.google.i18n.phonenumbers.Phonenumber;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.ResponseBody;
 
 @org.springframework.stereotype.Controller
 public class Controller {
 
-    @GetMapping ("/hello")
+    @GetMapping("/hello/{name}")
     @ResponseBody
-    public String hello(){
-        return  "Hello world";
+    public String hello(@PathVariable("name") String name) throws NumberParseException {
+
+
+        PhoneNumberUtil phoneUtil = PhoneNumberUtil.getInstance();
```

Comment:
masz statyczna instancje -> raz zrobic i trzymac w klasie

PHONUE_NUMBER_UTIL = PhoneNumberUti.getInstance();

@Get()
void... {
...UTIL.parse(

Reply:
w ogole to wg zadania mialo byc web osobno (hello/world) i formatowanie osobno
pom.xml glownej apki webowej powinien importowac modul z klaska ktora opakowuje libphonenumber